src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -28,17 +28,26 @@ class AlternativesDialog < UI::Dialog
     MIN_WIDTH = 60
     MIN_HEIGHT = 20
 
+    def initialize
+      @mock_slaves = {
+        :ed => %!<pre>editor.1.gz /usr/share/man/man1/ed.1.gz</pre>!,
```

Comment:
why not using more common `""` for string? In general I do not like all `%` constructors, as it is quite easy to overlook what exactly it create